mozilla-services/consvc-shepherd
# [SHEP-34] - Document required ads data access needed to run queries on BigQuery (BQ)

## References

[SHEP-34](https://mozilla-hub.atlassian.net/browse/SHEP-34)

## Problem Statement

We should explain that querying BigQuery (BQ) involves accessing sensitive ads data, which requires specific privileges to ensure proper access.

## Proposed Changes

- Explain that folks need to be granted [Cat 2 Ads access level](https://mozilla-hub.atlassian.net/wiki/spaces/MA1/pages/575897693/Ads+Sensitive+Data+Policy) in order to run the BQ script locally.

## Verification Steps

A step by step guide of how the PR reviewer can verify that changes are working correctly.

1. Read the docs

[SHEP-34]: https://mozilla-hub.atlassian.net/browse/SHEP-34?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
